src/gate_detect/test/gate-detect-test.cpp

```diff
@@ -0,0 +1,27 @@
+//
+// Created by Cameron on 03/04/18.
+//
```

Comment:
This default comment should be replaced with one with name, date, and description. See almost any `.cpp` or `.h` file in `sample_package` for an example of the correct formatting.

Reply:
Reformatted comments